src/command.js

```diff
@@ -1,65 +1,28 @@
-/**
- * Copyright (c) 2018-present The Palmer Group
- *
- * This source code is licensed under the MIT license found in the
- * LICENSE file in the root directory of this source tree.
- */
+import { COMPARE } from "./constants";
 
-import { MATCH, RECORD } from './constants';
+const matchImageSnapshotCommand = (subject, maybeName, maybeOptions) => {
```

Comment:
How about using `...args`.

Then we check the length:

```js
if (args.length === 3) {
    const [subject, name, options] = args;
} else if (args.length === 2) {
    const [subject, options] = args;
} ...
```

Reply:
Update: we checked this out and this doesn't fix the issue as we still need to type check the arguments anyway. So current version is probably good enough.